roles/satosa/templates/saml2_backend.yaml.j2

```diff
@@ -1,27 +1,45 @@
+---
 module: satosa.backends.saml2.SAMLBackend
 name: SamlSP
 config:
   sp_config:
     key_file: certs/backend.key
     cert_file: certs/backend.crt
-    organization: {display_name: {{ environment_string }} Proxy SP, name: {{ environment_string }} Proxy SP, url: 'https://{{ hostnames.meta }}/'}
+    organization:
+      display_name: {{ environment_string }} Proxy SP
+      name: {{ environment_string }} Proxy SP
+      url: 'https://{{ hostnames.meta }}/'
     contact_person:
```

Comment:
organization metadata should be fully configurable, too.  The values here "{{environment_strinf}} Proxy SP" etc can be the default values.

Reply:
Agree